src/synapse/workload.py

```diff
@@ -69,14 +69,28 @@ class ExecResult(typing.NamedTuple):
 
 
 def check_ready() -> typing.Dict:
-    """Return the Synapse container check.
+    """Return the Synapse container ready check.
 
     Returns:
         Dict: check object converted to its dict representation.
     """
     check = Check(CHECK_READY_NAME)
     check.override = "replace"
     check.level = "ready"
+    check.http = {"url": f"http://localhost:{SYNAPSE_PORT}/_synapse/admin/v1/server_version"}
```

Comment:
I think this is a constant in another file also?

Reply:
Good point, changed.